src/routes/LobbyRouter.ts

```diff
@@ -34,30 +36,33 @@ function generateLobbyName(
   type : ChannelType,
   owner : DiscordUser,
   guildUser : GuildUser,
+  textChat?: boolean,
 ) : string {
   let icon : string;
   if (type === ChannelType.Nojoin) icon = '🔐';
-  if (type === ChannelType.Mute) icon = '🙊';
+  else if (type === ChannelType.Mute) icon = '🙊';
   else icon = '🔊';
```

Comment:
sinds het toch een `let` is;

```ts
 let icon : string = '🔊';
  if (type === ChannelType.Nojoin) icon = '🔐';
  if (type === ChannelType.Mute) icon = '🙊';
```

Reply:
Is wel een slimme had ik niet aan gedacht